test-e2e/test/jestGlobalTeardown.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const del = require('del');
+
+module.exports = async function jestGlobalTeardown(_globalConfig) {
+  await del(
+    [path.join(global.__TEST_PATH__, '*'), '!' + path.join(global.__TEST_PATH__, 'esy')],
```

Comment:
Remove everything except for the esy store so that ocaml is cached for later runs.

Reply:
Nice! 💯